runtime/metricsview/astexpr.go

```diff
@@ -121,6 +121,8 @@ func (b *sqlExprBuilder) writeCondition(cond *Condition) error {
 		return b.writeJoinedExpressions(cond.Expressions, " OR ")
 	case OperatorAnd:
 		return b.writeJoinedExpressions(cond.Expressions, " AND ")
+	case OperatorEqNull, OperatorNeqNull:
+		return b.writeNullCondition(cond.Expressions, cond.Operator == OperatorEqNull)
```

Comment:
Parag actually just added handling of this here: 
https://github.com/rilldata/rill/blob/a370792226082e832932733ce352c7b2941578ef/runtime/metricsview/astexpr.go#L292-L296

It's in a slightly different way – just checking if `Eq` is used with a `nil` on one side. That avoids a new operator.

But looks like he didn't handle the `NOT` case, so you might want to add that :)